containers/Dockerfile.buildenv

```diff
@@ -23,6 +23,12 @@ RUN apt-get update -y \
     && apt-get install -y software-properties-common \
     && add-apt-repository ppa:ubuntu-toolchain-r/test
 
+# Add Kitware repo for cmake
```

Comment:
let's just divide python2 and python3 tests into separate Travis build targets and only test a single one as we need to make sure the boost python module is consistent with the spectre ones